mobile/lib/src/infrastructure/bluetooth_repository.dart

```diff
@@ -0,0 +1,125 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_blue/flutter_blue.dart';
+import 'package:polydodo/src/domain/acquisition_device/acquisition_device.dart';
+import 'package:polydodo/src/domain/acquisition_device/i_acquisition_device_repository.dart';
+import 'package:polydodo/src/domain/unique_id.dart';
+
+class BluetoothRepository implements IAcquisitionDeviceRepository {
+  static const String BLE_SERVICE = "fe84";
+  static const String BLE_RECEIVE = "2d30c082";
+  static const String BLE_SEND = "2d30c083";
+  static const startStreamChar = 'b';
+  static const stopStreamChar = 's';
+
+  BluetoothDevice selectedDevice;
+  BluetoothCharacteristic sendCharacteristic;
+  BluetoothCharacteristic receiveCharacteristic;
+
+  FlutterBlue flutterBlue;
+  StreamSubscription<List<ScanResult>> flutterSubscription;
+  static List<AcquisitionDevice> bluetoothPersistency = [];
+  final streamController = StreamController<List<AcquisitionDevice>>();
+
+  void initializeBluetooth() {
+    if (flutterSubscription == null) {
+      flutterBlue = FlutterBlue.instance;
```

Comment:
```suggestion
    flutterBlue ??= FlutterBlue.instance;
```

https://dart.dev/codelabs/dart-cheatsheet